Source/objects.cpp

```diff
@@ -4032,7 +4032,7 @@ bool OperateShrineSecluded(int pnum)
 	if (pnum != myplr)
 		return true;
 
-	std::fill(&automapview[0][0], &automapview[DMAXX][DMAXY], true);
+	std::fill(&automapview[0][0], &automapview[0][0] + sizeof(automapview), true);

```

Comment:
Wouldn't this be a simpler fix?
```suggestion
	std::fill(&automapview[0][0], &automapview[DMAXX - 1][DMAXY - 1], true);
```

Reply:
nah, to hell with these -1's 😅

Reply:
Using `sizeof` seems a lot more hacky to me.